diff --git a/utils/alsa-utils/Makefile b/utils/alsa-utils/Makefile
index 5e8d2bc..a80a1d5 100644
--- a/utils/alsa-utils/Makefile
+++ b/utils/alsa-utils/Makefile
@@ -56,7 +56,7 @@ define Package/alsa-utils/install
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/amixer $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/alsamixer $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/aplay $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/arecord $(1)/usr/bin/
+	$(LN) /usr/bin/aplay $(1)/usr/bin/arecord
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/alsactl $(1)/usr/sbin/
 
 	$(INSTALL_DIR) $(1)/usr/share/alsa/init
