From 30830d299441facee9ac0b325fcba811ad1b682d Mon Sep 17 00:00:00 2001
From: Joe Honold <mozzwald@gmail.com>
Date: Mon, 18 Jul 2016 11:51:39 -0500
Subject: [PATCH] alsa-utils fixup for salsa

---
 utils/alsa-utils/Makefile | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/utils/alsa-utils/Makefile b/utils/alsa-utils/Makefile
index 5e8d2bc..a1df55b 100644
--- a/utils/alsa-utils/Makefile
+++ b/utils/alsa-utils/Makefile
@@ -18,6 +18,9 @@ PKG_MD5SUM:=b9d6102fbbd0b68040bb77023ed30c0c
 PKG_INSTALL:=1
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>
 
+PKG_FIXUP:=autoreconf
+PKG_FIXUP:=gettext-version
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/alsa-utils
@@ -56,13 +58,7 @@ define Package/alsa-utils/install
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/amixer $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/alsamixer $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/aplay $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/arecord $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/alsactl $(1)/usr/sbin/
-
-	$(INSTALL_DIR) $(1)/usr/share/alsa/init
-	$(INSTALL_DATA) \
-	    $(PKG_INSTALL_DIR)/usr/share/alsa/init/* \
-	    $(1)/usr/share/alsa/init/
+	$(LN) /usr/bin/aplay $(1)/usr/bin/arecord
 endef
 
 define Package/alsa-utils-seq/install
-- 
2.1.4

